googleapiclient/http.py

```diff
@@ -468,7 +469,7 @@ def __init__(self, body, mimetype='application/octet-stream',
     resumable: bool, True if this is a resumable upload. False means upload
       in a single request.
     """
-    fd = StringIO.StringIO(body)
+    fd = BytesIO(body)
```

Comment:
I figured that I would keep it consistent and use `six` across the board for the IO's.  Is there a benefit for using `io` over `six`?